core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1820,7 +1820,11 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
         DBIO.from(for {
           resource <- {
             val projectOwnerPolicy = SamWorkspacePolicyNames.projectOwner -> SamPolicy(Set(projectOwnerPolicyEmail), Set.empty, Set(SamWorkspaceRoles.owner, SamWorkspaceRoles.projectOwner))
-            val ownerPolicy = SamWorkspacePolicyNames.owner -> SamPolicy(Set(WorkbenchEmail(userInfo.userEmail.value)), Set.empty, Set(SamWorkspaceRoles.owner))
+            val ownerPolicy =
+              if (workspaceRequest.onlyAddBillingProjectOwner.getOrElse(false))
+                SamWorkspacePolicyNames.owner -> SamPolicy(Set.empty, Set.empty, Set(SamWorkspaceRoles.owner))
+              else
+                SamWorkspacePolicyNames.owner -> SamPolicy(Set(WorkbenchEmail(userInfo.userEmail.value)), Set.empty, Set(SamWorkspaceRoles.owner))
```

Comment:
The only thing that differs in the 2 cases is the membership. I would have the condition create the right membership set. Also use curly braces please.

```
val ownerPolicyMembership = if (workspaceRequest.onlyAddBillingProjectOwner.getOrElse(false)) {
  Set.empty
} else {
  Set(WorkbenchEmail(userInfo.userEmail.value))
}
val ownerPolicy = SamWorkspacePolicyNames.owner -> SamPolicy(ownerPolicyMembership, Set.empty, Set(SamWorkspaceRoles.owner))
```

or maybe 
```
val ownerPolicyMember = workspaceRequest.onlyAddBillingProjectOwner.flatMap {
  case Some(true) => Option(WorkbenchEmail(userInfo.userEmail.value))
  case _ => None
}
val ownerPolicy = SamWorkspacePolicyNames.owner -> SamPolicy(ownerPolicyMember.toSet, Set.empty, Set(SamWorkspaceRoles.owner))
```